pkg/arvo/lib/chat-view.hoon

```diff
@@ -0,0 +1,50 @@
+/-  *chat-view, *rw-security
+|%
```

Comment:
Hmm, but with the `^?` removed, this would still expose the `*rw-security`. Wondering if there's some middle ground here where we can _selectively_ expose only the types (core) "native" to the lib...

Reply:
could we just cons the sur import and the library core together?

Reply:
Oh yeah, that seems to work.

You're gonna have to `=, chat-view` instead of importing as `*chat-view` though, which is bad-style use of `=,`. I guess the ends justify the means here.